test/python/algorithms/minimum_eigensolvers/test_vqe.py

```diff
@@ -107,6 +107,13 @@ def test_basic_aer_statevector(self, estimator):
         with self.subTest(msg="assert optimizer_result."):
             self.assertAlmostEqual(result.optimizer_result.fun, self.h2_energy, places=5)
 
+        with self.subTest(msg="assert eigenstate can be retrieved"):
+            sampler = Sampler()
+            ansatz = result.ansatz
+            ansatz.measure_all()
+            eigenstate_job = sampler.run(ansatz, result.optimal_point)
+            self.assertIsNotNone(eigenstate_job.result().quasi_dists)
```

Comment:
Would it be a better test to check that the energy with this ansatz and the optimal parameters matches the output energy of the VQE? 🙂

Reply:
I agree